app/routes/app.route.js

```diff
@@ -0,0 +1,52 @@
+import { Router } from 'express';
+import { handleError, handleSuccess } from '../services/feedbacks.service';
+import { getUser, getUserList, removeUser, updateUser, addUser } from '../services/users.service';
+
+export const router = Router();
+
+router.get('/list', async (req, res) =>  {
```

Comment:
Looks like you skipped user service layer. You make call of the data layer. In that case, user service will have only one call but in future there may be more of them.
I'd recommend move all methods related to storing data from `users.service` file to `models/user` and inside `user.service` should be only calls of them. And from controller you should call only user service. 
For example, you need to add logic to send email after updating user. In that case you will add the call of mail service inside user service not on controller. Controller will still have only call of user service method.

Reply:
{
  "url": "postgres://stfwrlrj:up7UlBBX50iObmzLK_QWu1t2zl-nm1YN@kandula.db.elephantsql.com:5432/stfwrlrj"
}